### DISCUSSION TITLE
cannot find plugins files on linux env

### DISCUSSION THREAD

**Discussion by eliranrefael** [skartknet/Iconic#29] [Q&A]
hey,
when im running the umbraco with the plugin using container on linux, and im trying to get to the edit section or to the picker itself, i get this: 
![image](https://user-images.githubusercontent.com/37138835/214606549-b14be58a-ce8c-4f4a-b9cc-1d1d7d45953f.png)

my theory is that the folders name aren't precise enough and it breaks because linux is case sensetive.
it should be written as "App_Plugins/Iconic/..."

what do you think?

thanks

**Comment by skartknet** [comment]
I'm not sure why this issue keeps coming back :). I thought we had already fixed this a couple of times. I'll take a another look.

**Reply by eliranrefael** [reply]
waiting for you response! you have an awesome plugin, i wish to work with it.
i can tell that in local env on my windows when im using iis express all good. when i run it with docker container on linux env it happens, i double checked that the files exist physically in the path and they do.

**Comment by skartknet** [comment]
Hi @eliranrefael , what version of the plugin are you using? The latest one in beta is 5.2.0-beta3

**Reply by eliranrefael** [reply]
hey, im using 5.1.0 which is labeled as lts, with umbraco 10.3.2

**Reply by skartknet** [reply]
I can see the routes seem to be fine on 5.2.0-beta3. Can you upgrade to that one and see if it works for you?

**Comment by skartknet** [comment]
Hi @eliranrefael , it has been a long time, but just wondering if I can close this discussion.

**Comment by eliranrefael** [comment]
Sure

בתאריך יום ג׳, 27 בפבר׳ 2024, 00:29, מאת Mario Lopez ‏<
***@***.***>:
